azblob/delete.go

```diff
@@ -0,0 +1,38 @@
+package azblob
+
+import (
+	"context"
+	"errors"
+
+	msazblob "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/datatrails/go-datatrails-common/logger"
+)
+
+// Put creates or replaces a blob
```

Comment:
minor: copy paste comment?

Reply:
yep, I fixed it yesterday. this diff is out of date. not sure why it isn't refreshing